app/utils/db.server.ts

```diff
@@ -16,6 +16,10 @@ export const prisma = remember('prisma', () => {
 			{ level: 'warn', emit: 'stdout' },
 		],
 	})
+
+	client.$queryRaw`PRAGMA journal_mode = WAL;`
```

Comment:
Can you help me understand why we're doing this? @benbjohnson, this will run as the first thing the prisma client does when it connects to the sqlite database. Do you see this as problematic or a reasonable workaround? Should we prevent this from running on read-only replicas?

Reply:
Yeah I'm not sure if this is the right thing to be doing. I added it because I figured we had to set the `journal_mode` in order to use WAL mode - I don't think it's on by default in LiteFS. [The comment that I linked above](https://github.com/prisma/prisma/issues/3303#issuecomment-1565192313) is part of a thread where Prisma sqlite users are discussing WAL mode and this seemed like a sensible option. I would definitely defer to @benjohnson.

Reply:
Ok I realize that this may be dumb because the command in the Dockerfile already does this. I guess my question would be for the local database and the test database? How can we make sure these are configured like the production database?

Reply:
This makes more sense I believe:
https://github.com/epicweb-dev/epic-stack/pull/621/commits/4ef05f524d548ff6e4f9ba5bfdd15fe86e27236d